### ISSUE TITLE
[FEATURE] More filters

### ISSUE THREAD

**Issue by Davidy22** [Davidy22/SkunkBooth#69]
Get creative, add more filters. See https://github.com/Davidy22/SkunkBooth/blob/main/skunkbooth/filters/matrix.py and https://github.com/Davidy22/SkunkBooth/blob/main/skunkbooth/filters/invert.py for reference.

**Comment by meenakshi0910** [comment]
Hi @Davidy22 , Can I work on this?

**Comment by Davidy22** [comment]
@meenakshi0910 Yeah sure, this is pretty much an indefinite desired addition, I just have it in ticket form to really highlight that this is a thing we want
